tensorflow/python/keras/layers/merge.py

```diff
@@ -235,9 +235,7 @@ class Add(_Merge):
       x1 = keras.layers.Dense(8, activation='relu')(input1)
       input2 = keras.layers.Input(shape=(32,))
       x2 = keras.layers.Dense(8, activation='relu')(input2)
-      added = keras.layers.Add()([x1, x2])  # equivalent to added =
-      keras.layers.add([x1, x2])
-
+      added = keras.layers.Add()([x1, x2])  # equivalent to added = keras.layers.add([x1, x2])
```

Comment:
Quoting from internal review:

> The comment should be moved to above this line to avoid the line-too-long issue.

> The code in the comment should be put inside backquotes

Reply:
I have made the changes. Thanks!